### PR TITLE
 Translate "Ruby 3.2.3 Released" (ko)

### PR DESCRIPTION
**Pull request by riseshia**
:link: https://github.com/ruby/www.ruby-lang.org/issues/2818

Actual diff: 00561be72c673564ed7dc6a3332ba2a45a8d6e4d

### CODE REVIEW

**Review by marocchino — APPROVED** [review]

**Comment by riseshia** [comment]
🙏

File: ko/news/_posts/2023-06-29-redos-in-uri-CVE-2023-36617.md [hunk 1]
```diff
@@ -40,4 +41,5 @@ NOTE: 이 문제는 [CVE-2023-28755](https://www.ruby-lang.org/en/news/2023/03/2
 
 ## 수정 이력
 
+* 2024-01-18 12:00:00 (UTC) Ruby 3.2에 새 권장 조치를 추가
```
Comment (yous) [inline ko/news/_posts/2023-06-29-redos-in-uri-CVE-2023-36617.md:44]: - Ruby 3.2에 → Ruby 3.2를 위한

**Review by yous — COMMENTED** [review]
LGTM, but one comment!

File: ko/news/_posts/2023-06-29-redos-in-uri-CVE-2023-36617.md (hunk 1 above)
Comment (riseshia) [reply ko/news/_posts/2023-06-29-redos-in-uri-CVE-2023-36617.md:44]: Thanks! -> https://github.com/ruby/www.ruby-lang.org/pull/3186

**Pull request merged**